### PR TITLE
41/web app UI styling for survey form

### PR DESCRIPTION
**Pull request by najwanan**
Ticket Description
UI changes for survey form and changes for responsiveness (med devices)
Screenshot 2022-12-06 at 1 07 24 PM

Description of Changes
- Changed background and buttons 
- Fixed margins and padding 
- Checked for responsiveness on all devices
Screenshot 2022-12-06 at 1 07 24 PM

<-- Brief description of the changes you made, files you touched, etc -->

Before and After for UI Updates
<-- You should have 2 images minimum, for desktop, 4 for mobile and desktop combined -->

Before:

After:
Screenshot 2022-12-06 at 1 03 59 PM
Screenshot 2022-12-06 at 1 04 07 PM

Screenshot 2022-12-06 at 1 03 33 PM

Screenshot 2022-12-06 at 1 04 16 PM

For PR Reviewer
 Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
 If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
 Does this file match the related tickets linked figma file, or does it pass the visual smell test?
 If this file contains javascript, does the javascript pass the smell test?
 If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: src/components/pages/SurveyPage.css [hunk 1]
```diff
@@ -31,21 +27,16 @@ h5 {
...
-  margin-right: 20em;
-}
 
 .actions {
   display: block;
   margin-left: 1em;
   margin-right: 1em;
 }
 
-.actions button {
```
Comment (Saige243) [inline src/components/pages/SurveyPage.css:46]: Are you using this? If not we can delete! (Unless there's a special future purpose for keeping it.) 😄

File: src/components/pages/SurveyPage.css [hunk 2]
```diff
@@ -88,7 +77,7 @@ h5 {
 }
 
 .question .horizontal div label {
-  padding:0;
+  /* padding:0; */
```
Comment (Saige243) [inline src/components/pages/SurveyPage.css:80]: same here!

**Review by Saige243 — APPROVED** [review]
Just take a peek at the comments of the items commented out. Not sure if you want to use those in the future, otherwise I think they can be scrapped! Other than that, good to go!

**Comment by najwanan** [comment]
reviewed and resolved merge conflicts. Merging PR

**Comment by najwanan** [comment]
Merging with testing errors per Daniel

**Pull request merged**